ci-operator/jobs/openstack-k8s-operators/heat-operator/openstack-k8s-operators-heat-operator-main-presubmits.yaml

```diff
@@ -53,14 +53,14 @@ presubmits:
     - ^main$
```

Comment:
We should set `always_run: false` in line 51 to avoid running both `build-deploy` and build-deploy-kuttl`, since the build and deploy steps would be duplicated

Reply:
@cescgina is correct.
We can avoid an extra job running here, setting to always_run: false, since build-deploy-kuttl already cover the build-deploy scenario.

Reply:
Changed this to false now.